config.go

```diff
@@ -10,112 +10,105 @@ const (
-	return lw.configDB.Set(userConfigBucketName, key, value)
+func (mw *MultiWallet) SaveUserConfigValue(key string, value interface{}) {
+	err := mw.configDB.Set(userConfigBucketName, key, value)
+	if err != nil {
+		log.Errorf("error reading config key: %s, value: %v", key, err)
+	}
 }
 
-func (lw *LibWallet) ReadUserConfigValue(key string, valueOut interface{}) error {
-	err := lw.configDB.Get(userConfigBucketName, key, valueOut)
-	if err != nil && err != storm.ErrNotFound {
-		return err
+func (mw *MultiWallet) ReadUserConfigValue(key string, valueOut interface{}, defaultValue interface{}) (interface{}, error) {
+	err := mw.configDB.Get(userConfigBucketName, key, valueOut)
+	if err == storm.ErrNotFound {
+		return defaultValue, nil
+	} else if err != nil {
+		log.Errorf("error reading config key: %s, value: %v", key, err)
+		return nil, err
 	}
-	return nil
+	return valueOut, nil
 }
 
-func (lw *LibWallet) SetBoolConfigValueForKey(key string, value bool) {
-	err := lw.SaveUserConfigValue(key, value)
-	if err != nil {
-		log.Errorf("error setting config value: %v", err)
-	}
+func (mw *MultiWallet) SetBoolConfigValueForKey(key string, value bool) {
+	mw.SaveUserConfigValue(key, value)
 }
 
-func (lw *LibWallet) SetDoubleConfigValueForKey(key string, value float64) {
-	err := lw.SaveUserConfigValue(key, value)
-	if err != nil {
-		log.Errorf("error setting config value: %v", err)
-	}
+func (mw *MultiWallet) SetDoubleConfigValueForKey(key string, value float64) {
+	mw.SaveUserConfigValue(key, value)
 }
 
-func (lw *LibWallet) SetIntConfigValueForKey(key string, value int) {
-	err := lw.SaveUserConfigValue(key, value)
-	if err != nil {
-		log.Errorf("error setting config value: %v", err)
-	}
+func (mw *MultiWallet) SetIntConfigValueForKey(key string, value int) {
+	mw.SaveUserConfigValue(key, value)
 }
 
-func (lw *LibWallet) SetLongConfigValueForKey(key string, value int64) {
-	err := lw.SaveUserConfigValue(key, value)
-	if err != nil {
-		log.Errorf("error setting config value: %v", err)
-	}
+func (mw *MultiWallet) SetInt32ConfigValueForKey(key string, value int32) {
+	mw.SaveUserConfigValue(key, value)
 }
 
-func (lw *LibWallet) SetStringConfigValueForKey(key, value string) {
-	err := lw.SaveUserConfigValue(key, value)
-	if err != nil {
-		log.Errorf("error setting config value: %v", err)
-	}
+func (mw *MultiWallet) SetLongConfigValueForKey(key string, value int64) {
+	mw.SaveUserConfigValue(key, value)
 }
 
-func (lw *LibWallet) ReadBoolConfigValueForKey(key string) (valueOut bool) {
-	err := lw.ReadUserConfigValue(key, &valueOut)
-	if err != nil {
-		log.Errorf("error reading config value: %v", err)
-	}
+func (mw *MultiWallet) SetStringConfigValueForKey(key, value string) {
+	mw.SaveUserConfigValue(key, value)
+}
+
+func (mw *MultiWallet) ReadBoolConfigValueForKey(key string, defaultValue bool) (valueOut bool) {
+	mw.ReadUserConfigValue(key, &valueOut, defaultValue)
```

Comment:
As mentioned in https://github.com/raedahgroup/dcrlibwallet/pull/57/files#r343517646, this should be
```
return (mw.ReadUserConfigValue(key, &valueOut, defaultValue)).(bool)
```